workspace/lc-test.js

```diff
@@ -1,7 +1,9 @@
 import { readFileSync } from "fs";
 
-export { assert, config } from "chai";
-export * as LC from "@codewars/lambda-calculus";
+import { assert, config } from "chai";
+import * as LC from "../src/lambda-calculus.js";
```

Comment:
This won't work because there's no `src/` in the image. `@codewars/lambda-calculus` is the package installed there.

To test without creating a new image, run `npm install` first in `workspace/` to get `workspace/node_modules/` then run `npx mocha`. If you have unreleased changes you want to test, release the package and update `package.json` before `npm install`. You can also do this temporarily locally to quickly test before releasing a new version, but don't commit the change.

Reply:
Oh of course 🤦‍♂️ I did plan to revert that but forgot